packages/deep-merge/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["plugin:@empathyco/x/all"]
```

Comment:
The `deep-merge.spec.ts` file is not read by eslint due to the `tsconfig.json`. We can fix it by simply moving it to `src/__tests__` like we do in x-components